whylog/config/parsers.py

```diff
@@ -57,3 +74,171 @@ def create_from_intent(cls, parser_intent):
     @classmethod
     def from_dao(cls, serialized_parser):
         return RegexParser(**serialized_parser)
+
+
+class ConcatenatedRegexParser(object):
```

Comment:
This class implements some kind of interface, so it should inherit AbstractSomething